<?xml version="1.0"?>
<launch>
    <param name="robot_description" command="cat $(find itr_rpc)/robots/4dofnonplanar.urdf" />
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find itr_rpc)/config/itr_rpc.rviz" />
    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    	<param name="use_gui" type="bool" value="True" />
    </node>
</launch>
